lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -3465,7 +3551,8 @@ def show_actions(mod) # :nodoc:
 
   def show_advanced_options(mod) # :nodoc:
     mod_opt = Serializer::ReadableText.dump_advanced_options(mod, '   ')
-    print("\nModule advanced options:\n\n#{mod_opt}\n") if (mod_opt and mod_opt.length > 0)
+    print("\nModule options (#{mod.fullname}):\n\n#{mod_opt}\n") if (mod_opt and mod_opt.length > 0)
```

Comment:
Yeah, copy-paste error on my part.  I was simply improving this line to show the module name (much like `options` and `show options` do), so this line should go and line 3555 should stay.